### ISSUE TITLE
router_route: out of destination slots, increase CONN_DESTS_SIZE in dispatcher.c

### ISSUE THREAD

**Issue by gmlwall** [grobian/carbon-c-relay#91]
relay output spams: 
router_route: out of destination slots, increase CONN_DESTS_SIZE in dispatcher.c

using latest version or carbon-c-relay

config file:

cluster graphite-new
    forward
      10.10.10.1:2013
      10.10.10.1:2023
      10.10.10.1:2033
      10.10.10.1:2043
      10.10.10.1:2053
      10.10.10.1:2063
      10.10.10.1:2073
      10.10.10.1:2083
      10.10.10.1:2093
      10.10.10.2:2013
      10.10.10.2:2023
      10.10.10.2:2033
      10.10.10.2:2043
      10.10.10.2:2053
      10.10.10.2:2063
      10.10.10.2:2073
      10.10.10.2:2083
      10.10.10.2:2093
      10.10.10.3:2013
      10.10.10.3:2023
      10.10.10.3:2033
      10.10.10.3:2043
      10.10.10.3:2053
      10.10.10.3:2063
      10.10.10.3:2073
      10.10.10.3:2083
      10.10.10.3:2093
      10.10.10.4:2013
      10.10.10.4:2023
      10.10.10.4:2033
      10.10.10.4:2043
      10.10.10.4:2053
      10.10.10.4:2063
      10.10.10.4:2073
      10.10.10.4:2083
      10.10.10.4:2093
  ;

match *
    send to graphite-new
    ;

**Comment by grobian** [comment]
Wow, that's a lot of servers!  Do you intend to scale this list up in the future?

**Comment by gmlwall** [comment]
I had intended to yes! That config was non-redundant (redundant would have doubled config, but in a new cluster) 
Also this config was not how i was going to implement the server clusters (this doesnt send metrics to the same cluster so there would be alot of holes in graphs) 
I only implemented this way as i was load testing a new cluster.

The actual config is more like;

cluster node_01
fnv1a_ch
10.10.10.1:2013
10.10.10.1:2023
10.10.10.1:2033
10.10.10.1:2043
10.10.10.1:2053
10.10.10.1:2063
10.10.10.1:2073
10.10.10.1:2083
10.10.10.1:2093
;

cluster node_02
fnv1a_ch
10.10.10.2:2013
10.10.10.2:2023
10.10.10.2:2033
10.10.10.2:2043
10.10.10.2:2053
10.10.10.2:2063
10.10.10.2:2073
10.10.10.2:2083
10.10.10.2:2093
;

etc;

4 carbon-c-relays routing to 4 servers, each running 9 carbon caches.

As a side note; if you ever have the chance, id love to discuss architecture with you. - the issue page may not be the best place :)

The cluster does about 3m metrics per min, with 4 backend server, the disk IO is maxed. From browsing your repos i see there is a reimplementation of graphite. 
Id also like to thank you for carbon-c-relay. The original carbon-relay exploded under the load we are sending.
